test/test_vectorstore_host.py

```diff
@@ -42,52 +40,76 @@ def prepare_splitted_documents(
 
 
 def setup_module(module):
-    pdf_path = "test/bc_summary.pdf"
-    splitted_docs = prepare_splitted_documents(pdf_path)
-    docname = string_to_base64(os.path.basename(pdf_path))
-    db = Milvus.from_documents(
-        documents=splitted_docs,
-        embedding=OpenAIEmbeddings(),
-        connection_args={"host": _HOST, "port": _PORT},
-    )
-    utility.create_alias(db.collection_name, f"{docname}_{db.collection_name}")
-    collection_names.append(db.collection_name)
-
+    pass
 
 def teardown_module(module):
```

Comment:
@fengsh27 where/when is the teardown performed? what happens if only one test function is called and not the entire module? should we use a pytest fixture to ensure proper teardown after the tests are run?

Reply:
> @fengsh27 where/when is the teardown performed? what happens if only one test function is called and not the entire module? should we use a pytest fixture to ensure proper teardown after the tests are run?

setup_module/teardown_module is module level. Per my understanding, teardown will be called once all the tests in this module are executed (even if only one test in this module is executed). Of course, pytest fixture is more powerful.

Reply:
It seems I understood your concerns. Actually, my intention is to ensure the thorough cleanup of the test database once all the tests within this module have been executed rather than performing cleanup after each test. Therefore, the `teardown_module` is exactly what I want.

Reply:
So we can't run the tests in isolation, since they partly depend on each other? See also #16.

Reply:
You're right, test_similarity_search does depends on test_store_embeddings. I'll fix it.